CMakeLists.txt

```diff
@@ -653,6 +657,9 @@ if(WITH_OPTIM)
         endif()
     elseif(BASEARCH_S360_FOUND)
         if(WITH_CRC32_VX)
```

Comment:
I wouldn't wrap check_s390_intrinsics in the if statement if we end up using that function for other things.

Reply:
I wasn't sure about that... I can unwrap it...